copyAbi.ts

```diff
@@ -0,0 +1,237 @@
+import fs from 'fs';
```

Comment:
What is the purpose of this?  Can it be accomplished using [hardhat-abi-exporter](https://github.com/ItsNickBarry/hardhat-abi-exporter)?

Reply:
If not, let's move this to a post-compilation hook.  I don't want to have to modify this list every time a contract is added or removed.

Reply:
Note that ABIs are already exported to `abi` directory and published as `@solidstate/abi`.

Reply:
It requires the full artifacts.
I just found out about this hardhat plugin : https://www.npmjs.com/package/hardhat-typechain
I'll replace with that